examples/darwin-framework-tool/commands/common/CHIPToolKeypair.mm

```diff
@@ -79,9 +79,27 @@ - (SecKeyRef)publicKey
         };
         _mPublicKey = SecKeyCreateWithData((__bridge CFDataRef) publicKeyNSData, (__bridge CFDictionaryRef) attributes, nullptr);
     }
+}
+
+- (SecKeyRef)publicKey
```

Comment:
Seems fine, but why do we need this bit still here?

Reply:
Looks like cruft from when I was just trying to get things compiling.  I'll remove it and unwind the extraction of `_populatePublicKeyIfNecessary`.